c-dependencies/js-compute-runtime/builtins/dictionary.cpp

```diff
@@ -11,13 +11,33 @@ DictionaryHandle Dictionary::dictionary_handle(JSObject *obj) {
 bool Dictionary::get(JSContext *cx, unsigned argc, JS::Value *vp) {
   METHOD_HEADER(1)
 
+  JS::HandleValue name_arg = args.get(0);
+
   size_t name_len;
-  JS::UniqueChars name = encode(cx, args[0], &name_len);
+  // Convert into a String following https://tc39.es/ecma262/#sec-tostring
+  JS::UniqueChars name_chars = encode(cx, name_arg, &name_len);
+  if (!name_chars) {
+    return false;
+  }
+
+  // trim whitespace from key
```

Comment:
Was this a TODO? It looks like it's only checking `name_len` now.

Reply:
🙈 this was just a not to myself and not a todo ha
```suggestion
```